### PR TITLE
Updating LTS docs

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

Once v6 is released we still want to update docs around v5 if necessary. This patch ensures that all releases on v5 branch also get reflected in the v5 docs at (https://v5.webdriver.io). I am currently creating a new AWS distribution and will do the same for v4. The page will look as follows:

Screenshot 2019-12-31 at 13 46 55

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

This PR should be merged last once we actually want to do the release.

### CODE REVIEW

File: docs/Versions.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+|----|--------------------|--------------------------------------------------------------------------------------|-----------------|
+| v6 | [Documentation](/) | [Release Notes](https://github.com/webdriverio/webdriverio/blob/master/CHANGELOG.md) | __Stable__ |
+
+## Past Versions
+
+Here you can find documentation for previous versions of WebdriverIO.
+
+| | | | |
+|----|-------------------------------------------|----------------------------------------------------------------------------------------------|-----|
+| v5 | [Documentation](https://v5.webdriver.io/) | [Release Notes](https://github.com/webdriverio/webdriverio/blob/v5/CHANGELOG.md#changelog) | __LTS__ (until January 2021) |
```
Comment (erwinheitzman) [inline docs/Versions.md]: We don't have a v5 link yet, should we fix this first or wait till v6?

File: docs/Versions.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+title: WebdriverIO Versions
+---
+
+The project team releases new major versions roughly on a yearly cadence. LTS release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 12 months until a new major release is made.
+
+## Current version (Stable)
+
+| | | | |
+|----|--------------------|--------------------------------------------------------------------------------------|-----------------|
+| v6 | [Documentation](/) | [Release Notes](https://github.com/webdriverio/webdriverio/blob/master/CHANGELOG.md) | __Stable__ |
```
[the remark this answers is not in the document]
Comment (christian-bromann) [reply docs/Versions.md]: Nope, it links to https://webdriver.io which will be the v6 docs.

File: docs/Versions.md (hunk 1 above)
Comment (christian-bromann) [reply docs/Versions.md]: yeah , this was a configuration issue which I just fixed.

**Comment by christian-bromann** [comment]
@erwinheitzman any other comments?

**Comment by christian-bromann** [comment]
DON'T MERGE THIS PR, until v6 has been released!

**Pull request merged**